annotationURL.html

```diff
@@ -0,0 +1,126 @@
+<!doctype html>
+<html>
+    <head>
+        <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+        <meta content="utf-8" http-equiv="encoding">
+        <link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">
+        <link rel="stylesheet" type="text/css" href="css/annotation.css">
+        <link rel="stylesheet" type="text/css" href="css/font-awesome.min.css">
+        <title>
+            Annotation of an URL if any. 
+        </title>
+    </head>
+    <body>
+        <table width="100%" style="max-height: 200px; ">
+            <tr class="spaceUnder">
+                <td style="text-align: center;">
+                    <div id ="horizontal_line" style="margin-bottom: 5px"></div>
+                        <div class='container' style="background: white">
+                            <img class='resize_fit_center'
+                            src='images/logo.gif' />
+                        </div>
+                    <div id="horizontal_line" style="margin-top:3px"></div>
+                </td>
+            </tr>
+        </table>
+        <div>
+            <h2 style="text-align: center;"><strong>Annotations of Current URL</strong><br><span style="font-size: 50%;font-family:sans-serif"> From <a style="cursor: pointer;" href="http://hypothes.is/">Hypothes.is</a></span></h2>
+        </div>
+        <div class="container" id="container-whole">
+
+        </div>
+        <div class="row" id="row-contain" style="    display: flex;
+        align-items: stretch;
+        justify-content: center;" >
+            <div class="col-sm-6" style="width: 600px">
+                <div class="box">
+                    <div class="top-div" id="top-div-contain">
+                        <strong>
+                            <span id="userinfo">
+
+                            </span>
+                        </strong>
+                        <span id="date">
+
+                        </span>
+                    </div>
+                    <div class="source" id="source-">
+                        <span class="glyphicon glyphicon-globe" id="public"></span>
+                        <span style="font-weight: 150%;color: #818181">Public on</span>
+                        <span class="title" id="title-contain" style="color: red">
+
+                        </span>
+                        <span id="source-contain" style="word-wrap: break-word"> 
+                        </span>
+                    </div>
+                    <div class="target-selector-exact" id="target-selector-exact-contain">
+                    </div>
+                    <div class="text" id="text-contain">
+                    </div>
+                    <div id="links">
+                    </div>
+                </div>
+            </div>
+        </div>
+    </body>
+</html>
+<script src="scripts/annotationURL.js"></script>
+<style>
+body{
```

Comment:
Again, since there is an `annotation.css` file, why do you have this CSS here? Isn't it better to put all the CSS together?

Idea: You could use a common prefix e.g. all annotation css class could be named like `annot-title`, `annot-line` etc. to be more clear.